### ISSUE TITLE
No date translation in single post view (at least in french)

### ISSUE THREAD

**Issue by Globulle** [diaspora/diaspora#7645]
As said in the title:
![image](https://user-images.githubusercontent.com/11587267/31705209-bb9931e8-b3e4-11e7-9844-ab7b7505752a.png)

I haven't been able to found the segment on [Webtranslateit](https://webtranslateit.com/en/projects/7017-website).

EDIT : 
That's very strange: by the time I wrote this issue, the time stamp has changed on the page I have shown:
![image](https://user-images.githubusercontent.com/11587267/31705381-6f566de0-b3e5-11e7-99f5-0771c9391ac9.png)

**Comment by denschub** [comment]
Check this: https://webtranslateit.com/en/projects/3020-diaspora/locales/en..fr/strings?utf8=%E2%9C%93&object=&s=javascripts.timeago

**Comment by Globulle** [comment]
It seems this is not a translation issue: the segments exist in French, and they do appear after some minutes. I've tested again right now and the same scenario happen : 

- Click on the date of a post (in French) in my stream to get SPV : the date is given in English
- After a few minutes, the date appears again in French

**Comment by Flaburgan** [comment]
Do you have a JavaScript blocker, or something like that?

**Comment by Flaburgan** [comment]
Hm, I do reproduce the problem using Firefox Nightly...

**Comment by SuperTux88** [comment]
Yes, that's a bug in the frontend on the single-post-view and it's with all languages, not only french. It's probably a race-condition with timeago, so the language isn't set yet when it runs first, and it's then there when it updates.
